client/src/ui/components/friends/ConversationList.tsx

```diff
@@ -46,7 +45,7 @@ const ConversationList: React.FC<ConversationListProps> = ({
     if (addFriend == "") {
       showErrorToast("Username not found.");
     } else {
-      showSuccessToast("Friend request sent.");
+      showSuccessToast(`Friend request sent to ${addFriend}.`);
```

Comment:
`addFriend` is an odd name for the friend you're trying to add since it sounds like a function cause it's a verb, I would change the variable to something like `friendToAddName`